src/components/shared/ButtonWithPanel/index.js

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+// This implements a Button that triggers a panel. The panel will have a small
+// arrow pointing towards the button, which is implemented in ArrowPanel.
+//
+// All public API is exposed from this file. Other components should never be
+// used directly.
+//
+// Here is a simple example:
+//
+// import { ButtonWithPanel } from // 'firefox-profiler/components/ButtonWithPanel';
+// ...
+// <ButtonWithPanel
+//   className="MyButtonWithPanel"
+//   buttonClassName="MyPanelButton"
+//   panelClassName="MyPanel"
+//   label="Click me!"
+//   panelContent={<>
+//     <p>We explain lots of useful things here.</p>
+//     <p>If you want to know more <a href='/'>click here</a>.</p>
+//   </>}
+// />
+
+export { ButtonWithPanel } from './ButtonWithPanel';
```

Comment:
This single re-export is very mysterious to me, I would very much prefer to have the contents of ButtonWithPanel.js in this file. I think this approach only adds more complexity to the implementation.

Reply:
mmm yeah previously I had `ConfirmDialog` exported too, and I believe that was making things more understandable. But now it's not the case anymore, so I can as well do what you suggest.